src/util/Fd.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "Fd.h"
+
```

Comment:
wondering if you should explicitely have `#include <fcntl.h>` in order to call fcntl (portability)